src/components/panels/BountyPanel.vue

```diff
@@ -67,7 +67,7 @@ export default {
   props: ['syndicate', 'type'],
   computed: {
     headertext() {
-      return `${this.type} ${this.$t('bounty.header')}`;
+      return this.$t('bounty.header', {type: this.$t(`timer.${this.type.toLowerCase().replace(/\s/gi, '-')}`)});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace `type:·this.$t(`timer.${this.type.toLowerCase().replace(/\s/gi,·'-')}`)` with `·type:·this.$t(`timer.${this.type.toLowerCase().replace(/\s/gi,·'-')}`)·`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5335813)